openstack/swift/templates/_utils.tpl

```diff
@@ -165,6 +165,10 @@ location / {
     proxy_request_buffering off;
     # accept large PUT requests (5 GiB is the limit for a single object in Swift)
     client_max_body_size    5g;
+{{ if .Values.client_timeout -}}
```

Comment:
That will not work. The values are not passed to the definition and are not accessible.